ios/src/TGMapViewController.h

```diff
@@ -56,6 +56,7 @@ struct TileID;
 @property (assign, nonatomic) float zoom;
 @property (assign, nonatomic) float rotation;
 @property (assign, nonatomic) float tilt;
+@property (readonly, nonatomic) CGFloat pixelScale;
```

Comment:
Removed that one, figured the user won't have to deal with that from the discussion we had in the meeting, so it's always in logical pixel from the public interface.